examples/help_categories.py

```diff
@@ -4,7 +4,7 @@
 A sample application for tagging categories on commands.
 """
 
-from cmd2 import Cmd, categorize, __version__, with_argparser, with_category
+from cmd2.cmd2 import Cmd, categorize, __version__, with_argparser, with_category
 import argparse
```

Comment:
``argparse`` is built-in to Python, so should generally be imported at the top before 3rd-party or custom imports.